app/message-receiver.js

```diff
@@ -0,0 +1,35 @@
+const { ReceiveMode } = require('@azure/service-bus')
+const MessageBase = require('./message-base')
+const logTraceMessage = require('./log-trace-message')
```

Comment:
You probably need to change the name to something like 'app-insights-....' as in the future we may want to track excpetions, events, etc.

Reply:
Agreed, see other comment.